spec/factories/state_factory_ override.rb

```diff
@@ -0,0 +1,28 @@
+require 'carmen'
+require 'spree/testing_support/factories/country_factory'
+
+FactoryGirl.modify do
```

Comment:
I am not sure how I feel about an extension modifying a factory Solidus provides. Can we instead add a new factory that uses the base `:state` factory as a parent? If you are explicitly calling this, that should be sufficient. If this is getting called from another factory, then that might not be a viable option. 
I want to avoid someone's tests starting to fail once they add this extension, because they rely on the factory in their version of Solidus.